R/execution.r

```diff
@@ -215,9 +215,9 @@ execute = function(request) {
     err <<- list()
     
     # shade base::quit
-    assign('quit', .self$quit, envir = .GlobalEnv)
-    assign('q',    .self$quit, envir = .GlobalEnv)
-    
+    .irk.add_to_user_searchpath('quit', .self$quit)
```

Comment:
I think there are a lot of things which are static between code executions, one could argue that these belong to the init method of the executer. -> later PR to cleanup that?